threeML/plugins/OGIP/eventlist.py

```diff
@@ -1062,12 +1113,10 @@ def __init__(self, arrival_times, energies, n_channels, start_time=None, stop_ti
 
             self._dead_time = None
 
-
-
-    def _exposure_over_interval(self, tmin, tmax):
+    def exposure_over_interval(self, tmin, tmax):
```

Comment:
It is not supposed to be called by the user, but it is called from an EventList object externally. Therefore, I believe that it should be public. (unless I misinterpret the public/private concept in python) I have added a detailed doctstring.

Note, EventList is not exposed to the user.